circus-train-avro/src/main/java/com/hotels/bdp/circustrain/avro/util/SchemaCopier.java

```diff
@@ -75,12 +75,14 @@ public Path copy(String source, String destination) {
   }
 
   private void copyToLocal(Path sourceLocation, Path localLocation) {
-    FileSystem sourceFileSystem;
     try {
+      LOG.info("Attempting to copy from {} to {}", sourceLocation, localLocation);
+      FileSystem sourceFileSystem;
```

Comment:
I guess you can combine the this and the next line

Reply:
+1